.github/workflows/update-generated.yaml

```diff
@@ -19,6 +19,7 @@ jobs:
       - name: Checkout
         uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac #v4.0.0
         with:
+          token: "${{ secrets.EKSCTLBOT_TOKEN }}"
```

Comment:
do we need double quotes here? its not used in line 56 🤔

Reply:
```suggestion
          token: ${{ secrets.EKSCTLBOT_TOKEN }}
```